deployer/infra_components/cluster.py

```diff
@@ -3,11 +3,35 @@
 import subprocess
 import tempfile
 from contextlib import contextmanager
-from pathlib import Path
 
-from .file_acquisition import get_decrypted_file, get_decrypted_files
+from deployer.utils.file_acquisition import (
+    HELM_CHARTS_DIR,
+    get_decrypted_file,
+    get_decrypted_files,
+)
+from deployer.utils.rendering import print_colour
+
 from .hub import Hub
-from .utils import print_colour, unset_env_vars
+
+
+@contextmanager
+def unset_env_vars(vars):
```

Comment:
I feel this is generic enough it should continue to be in something under `utils` rather than here.